### PR TITLE
Implement Coinfield

### PR DESCRIPTION
**Pull request by danielchengml**
- Implemented Pairs, Tickers and updated README for Coinfield 
- Related to Issue #544
- [ x ] I have added Specs
- [ x ] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [ x ] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

File: spec/exchanges/coinfield/market_spec.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+RSpec.describe Cryptoexchange::Exchanges::Coinfield::Market do
+  it { expect(described_class::NAME).to eq 'coinfield' }
+  it { expect(described_class::API_URL).to eq 'https://platform.coinfield.com/api/v2/' }
+end
```
[the remark this answers is not in the document]
Comment (danielchengml) [reply spec/exchanges/coinfield/market_spec.rb]: @tmlee Space where? I can't seem to spot it..

File: README.md [hunk 2]
```diff
@@ -22,188 +22,192 @@ Or install it yourself as:
 
 ## Exchanges Supported
 
-| Exchange          | Ticker  | Order Book | Trade   | Account | Market List |
```
[the remark this answers is not in the document]
Comment (danielchengml) [reply README.md]: Yeah, as mentioned in the exrates one, auto formatter issue. Have disabled it.

**Comment by danielchengml** [comment]
Updated README Format- Reverted to original format

**Comment by danielchengml** [comment]
Updated README format- revert to correct format

**Comment by danielchengml** [comment]
@tmlee Travis failed but details shows that Braziliex integration spec fails. And my commit doesn't have brazilliex

File: spec/exchanges/coinfield/market_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmlee) [reply spec/exchanges/coinfield/market_spec.rb]: Add a new line at the end of the file if possible before commiting to git
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

**Comment by danielchengml** [comment]
@tmlee Updated newline. Thanks for the pointer.

**Pull request merged**